fastpass/src/main/scala/scala/meta/internal/fastpass/bazelbuild/Bazel.scala

```diff
@@ -187,20 +183,24 @@ class Bazel(bazelPath: Path, cwd: Path) {
         "--output=textproto"
       )
 
-      val code = {
-        val errByteArrayOutputStream = new ByteArrayOutputStream
+      val errByteArrayOutputStream = new ByteArrayOutputStream
+
+      @tailrec
+      def runBazelAqueryCommand(retries: Int): Int = {
```

Comment:
I did the invocation of the run command during one `ProgressConsole` in order to not duplicate the displayed progress bars in console (now only one progress bar for `Querying action graph` is visible even if the aquery command returns an error 3 times  - after thrid fail information about failed is displayed)